server/sonar-server/src/main/java/org/sonar/server/language/ws/ListAction.java

```diff
@@ -80,7 +80,7 @@ void define(WebService.NewController controller) {
   }
 
   private Collection<Language> listMatchingLanguages(@Nullable String query, int pageSize) {
```

Comment:
since you are using `Pattern.quote` on it, you should rename parameter `query` to `stringLitteral`
the point being to highlight that adding any wildcard or such special character will be ignored